content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -4,64 +4,78 @@ Set up your barcode scanner
 
 .. _barcode/setup/hardware:
 
-Getting started with barcode scanning in Odoo is fairly easy. Yet, a
-good user experience relies on an appropriate hardware setup. This guide
-will help you through the task of choosing and configuring the barcode
-scanner.
-
-Find the barcode scanner that suits your needs
-==============================================
-
-The 3 recommended type of barcode scanners to work with the Odoo
-**Inventory** and **Barcode Scanning** apps are the **USB scanner**, **the bluetooth
-scanner** and the **mobile computer scanner**.
-
-.. image:: hardware/hardware02.png
-    :align: center
-
--   If you scan products at a computer location, the **USB scanner** is the
-    way to go. Simply plug it in the computer to start scanning. Just
-    make sure when you buy it that the scanner is compatible with
-    your keyboard layout or can be configured to be so.
-
--   The **bluetooth scanner** can be paired with a smartphone or a tablet and
-    is a good choice if you want to be mobile but don't need a big
-    investment. An approach is to log in Odoo on you smartphone, pair
-    the bluetooth scanner with the smartphone and work in the
-    warehouse with the possibility to check your smartphone
-    from time to time and use the software 'manually'.
-
--   For heavy use, the **mobile computer scanner** is the handiest solution.
-    It consists of a small computer with a built-in barcode scanner.
-    This one can turn out to be a very productive solution, however
-    you need to make sure that is is capable of running Odoo smoothly.
-    The most recent models using Android + Google Chrome or Windows +
-    Internet Explorer Mobile should do the job. However, due to the
-    variety of models and configurations on the market, it is
-    essential to test it first.
-
-Configure your barcode scanner
-==============================
+Follow this guide to choose and set up a barcode scanner compatible with Odoo's *Inventory* and
+*Barcode* apps.
+
+.. image:: hardware/barcode-scanner.png
```

Comment:
alternatively you could use a more descriptive visual directive like [figure](https://pandemic-overview.readthedocs.io/en/latest/myGuides/reStructuredText-Images-and-Figures-Examples.html#figure) here, which would include a written caption as to what we're looking at.